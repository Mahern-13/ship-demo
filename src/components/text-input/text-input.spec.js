import React from "react";
import { shallow } from "enzyme";
import TextInput from "./index";

describe("Checkbox", () => {
  it("renders the component", () => {
    const wrapper = shallow(<TextInput id={1} />);
    expect(wrapper).toMatchSnapshot();
  });

  it("renders with a preset value", () => {
    const wrapper = shallow(<TextInput id={1} value="test" />);
    expect(wrapper).toMatchSnapshot();
  });

  it("renders with a preset label", () => {
    const wrapper = shallow(<TextInput id={1} active={true} label="Test" />);
    expect(wrapper).toMatchSnapshot();
  });
});
