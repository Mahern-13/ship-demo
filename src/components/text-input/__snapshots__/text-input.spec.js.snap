// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox renders the component 1`] = `ShallowWrapper {}`;

exports[`Checkbox renders with a preset label 1`] = `ShallowWrapper {}`;

exports[`Checkbox renders with a preset value 1`] = `ShallowWrapper {}`;
